jujugui/static/gui/src/app/assets/css/inspector/_vars.scss

```diff
@@ -1,9 +1,10 @@
 $inspector-background-color: #221e1c;
```

Comment:
Yep, that colour only gets used in a couple of places (like the onboarding panels) and not for the inspector now.